test/build.prop/astro_55r.log

```diff
@@ -0,0 +1,31 @@
+# begin build properties
+# autogenerated by buildinfo.sh
```

Comment:
where is this file?

Reply:
It should be located under /system/build.props. However, I can't read it without root, so I pulled the template file and populated this by running adb shell getprop. I can remove this file, if desired.